### PR TITLE
Fix for Behaviors Overview Widget and Show All translation ( fixes #398 )

### LINKED ISSUE #398: No translations for behavior widget
[//]: # (
. Note: for support questions, please use Stackoverflow or Gitter**. 
. This repository's issues are reserved for feature requests and bug reports.
.
. In case of any problems with Allure Jenkins plugin** please use the following repository 
. to create an issue: https://github.com/jenkinsci/allure-plugin/issues
.
. Make sure you have a clear name for your issue. The name should start with a capital 
. letter and no dot is required in the end of the sentence. An example of good issue names:
.
. - The report is broken in IE11
. - Add an ability to disable default plugins
. - Support emoji in test descriptions
)

#### I'm submitting a ... 
  - [x] bug report
  - [ ] feature request
  - [ ] support request => Please do not submit support request here, see note at the top of this template.

#### What is the current behavior?
No translations for behavior widget
image_20170613151711417

#### If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem
* open overview page with many feature

#### Please tell us about your environment:

- Allure version:        2.1.1
- Test framework:        any
- Allure adaptor:        any
- Generate report using: any

#### Other information 

[//]: # (
. e.g. detailed explanation, stacktraces, related issues, suggestions 
. how to fix, links for us to have more context, eg. Stackoverflow, Gitter etc
)

### PR DESCRIPTION
**Pull request by JMoravec**
### Context
Originally I had set to fix the show all button on the behaviors overview widget. However, while attempting to test the change, I realized there was another bug, where it appears the behavior widget didn't have the correct `addWidget` call anymore, nor did it have an `initialize` call on the base widget. This is also now fixed. 

screen shot 2017-06-30 at 2 14 27 pm

As I'm not familiar much with this code base, please let me know if this solution isn't correct or the best way. Thanks!

#### Checklist
- [x] [Sign Allure CLA][cla]
- [ ] Provide unit tests

[cla]: https://cla-assistant.io/accept/allure-framework/allure2

### CODE REVIEW

**Comment by JMoravec** [comment]
fixes #398

File: allure-generator/src/main/javascript/components/widget-status/WidgetStatusView.hbs [hunk 1]
```diff
@@ -15,7 +15,7 @@
         {{#if (gt total items.length)}}
             <a class="table__row" href="#{{baseUrl}}">
                 <div class="table__col table__col_center">
-                    {{showAllText}}
+                    {{t showAllText}}
```
Comment (baev) [inline allure-generator/src/main/javascript/components/widget-status/WidgetStatusView.hbs]: `{{t 'component.widgetStatus.showAll'}}`

and add translations to https://github.com/allure-framework/allure2/tree/master/allure-generator/src/main/javascript/translations

Comment (baev) [inline allure-generator/src/main/javascript/components/widget-status/WidgetStatusView.hbs]: sorry, my fault

**Comment by baev** [comment]
@JMoravec thanks!

**Pull request merged**